index.html

```diff
@@ -1,414 +1,358 @@
+        <ul class="menu-items">
+          <li><a onclick="myFunction1()" href="#Portfolio">Portfolio</a></li>
+          <li><a onclick="myFunction1()" href="#About">About</a></li>
+          <li><a onclick="myFunction1()" href="#Contact">Contact</a></li>
+        </ul>
+      </div>
     </section>
     <section id="firstPage" class="heading">
-        <header>
-            <nav class="site-nav">
-                <a href="#">
-                    <img class="my-logo" src="images/my-logo.png" alt="logo">
-                </a>
-                <a href="#">
-                    <img onclick="myFunction()" class="menu" src="images/mymenu.png" alt="union">
-                </a>
-                <ul>
-                    <li><a href="#">Portolio</a></li>
-                    <li><a href="#">About</a></li>
-                    <li><a href="#">Contact</a></li>
-                </ul>
-            </nav>
-        </header>
-        <section class="main-section">
-            <br>
-            <h1 class="main-text">I’m Godanah <br> Glad to see you!</h1>
-            <p class="content">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-            <br>
-            <p class="p-page1">LET'S CONNECT</p>
-            <br>
-            <div class="contact">
-                <ul class="list">
-                    <li><a href="#"><img src="images/vector-1.png" alt="vector1"></a></li>
-                    <li><a href="#"><img src="images/vector-2.png" alt="vector2"></a></li>
-                    <li><a href="#"><img src="images/vector-3.png" alt="vector3"></a></li>
-                    <li><a href="#"><img src="images/vector-4.png" alt="vector4"></a></li>
-                    <li><a href="#"><img src="images/vector-5.png" alt="vector5"></a></li>
-                </ul>
-            </div>
-        </section>
-    </section>
-<section class="projects">
-    <section id="Portfolio" class="card">
-        <section class="profile-picture for-cards">
-            <img class="card-img" src="images/snapshot-profile-1.png" alt="Snapshot-profile">
-            <img class="card-img1" src="images/card-img1.png" alt="snapshot-profile">
-        </section>
-        <section class="tonic for-cards">
-            <h1>Tonic</h1>
-            <div>
-                <ul class="list">
-                    <li><p>CANOPY</p></li>
-                    <li><img src="images/counter-1.png" alt="counter1"></li>
-                    <li><p>Back End Dev</p></li>
-                    <li><img src="images/counter-2.png" alt="counter2"></li>
-                    <li><p>2015</p></li>
-                </ul>
-            </div>
-            <p class="content">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-            <br>
-            <div class="buttons">
-                <ul class="list">
```

Comment:
The project section is hidden on medium and small viewports.  So I am unable to review the pop-up details. Kindly see the below screenshot for reference.
![missing](https://user-images.githubusercontent.com/35504149/132006701-d32f4ea2-2468-4d4e-9f59-d334efb63945.png)